---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}
[Download](/files/shashi_resume.pdf)

## Current Role

### Staff Scientist, Warren Lab, Clinical Research Division, Fred Hutchinson Cancer Center (April 2022 - Current)
* Investigating the role of T-cells within the tumor micro-environment of individuals with epidemic and endemic KS from sub-Saharan Africa. (Manuscript in preparation)
* Identifying the global diversity of Epstein Barr Virus (EBV) and understanding its impact on the immune response towards EBV and EBV-associated malignancies. (Manuscript submitted)
* Explored the impact of HLA Class II genotype on metastatic non-small cell lung cancer (mNSCLC) patients undergoing immune checkpoint inhibitor (ICI) therapy. ([Nature Scientific Reports](https://www.nature.com/articles/s41598-023-48546-y))

## Research Interests

* T-Cell Specificity Prediction using Large Language Models
* 3D Structure Prediction of T-Cell Receptor-pMHC Complexes
* Multi-Instance Learning for T-Cell Repertoire Level Characteristic Identification
* Epitope Hotspot Identification in Kaposi Sarcoma Herpesvirus (KSHV)

## Work experience

### Bioinformatics Analyst, Warren Lab, Clinical Research Division, Fred Hutchinson Cancer Center. (October 2019 - April 2022)
  * Investigating the impact of extended antiretroviral therapy (ART) on the T-cell receptor \(\beta\) -chain (TRB) repertoire in individuals living with HIV. ([Frontiers in Immunology](https://www.frontiersin.org/articles/10.3389/fimmu.2022.879190/full))
  * Understanding the tumor micro-environment of Kaposi Sarcoma using single-cell RNA-Seq.
  * Developing tools and pipelines for [AIRR-Seq](https://github.com/shashidhar22/LymphoSeq2), [scRNA-Seq](https://github.com/shashidhar22/scrna_seq_pipelines) analysis, and [viral genome assembly](https://github.com/shashidhar22/viral_genomics).

### SWEP Graduate Researcher, Malaria Branch, Centers for Disease Control and Prevention, Malaria Branch, Centers for Disease Control and Prevention. (January 2018 - August 2018)
  * Developing a scalable NGS workflow for the identification of variations associated with drug resistance in \textit{Plasmodium falciparum}. ([Antimicrobial agents and chemotherapy](https://aac.asm.org/content/62/4/e02474-17.short) and [bioRxiv](https://www.biorxiv.org/content/10.1101/323535v3.abstract))
  * Constructed the first ever complete genome of the simian infecting malaria parasite \textit{Plasmodium brasilianum}. ([Nature Scientific Reports](https://www.nature.com/articles/s41598-022-20706-6))

### Bioinformatics Intern, New York Genome Center. (June 2015 - August 2015)
  * Developing a framework for Bisulfite-Seq analysis.
  * Benchmarking tools for variant identification from Bisulfite-Seq data.
  * Developing a tool for the detection of allele-specific methylation from Bisulfite-Seq data.

### Junior Bioinformatics Analyst, Interpretomics India Pvt Ltd. (July 2013 - May 2014)
  * Design and implementation of a Python based pipeline for Whole Genome/Exome Sequencing, RNA-Seq, ChIP-Seq, microRNA-Seq, and HLA-Typing for NGS-based cloud platform called iOMICS.
  * Developing a tool for the detection of the complete spectrum of variations in WGS/WES data. ([PLOS One](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0123569))

## Publications
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
## Talks
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
## Teaching
  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
## Service and leadership
* Currently signed in to 43 different slack teams

