---
title: "Next-generation sequencing and bioinformatics protocol for malaria drug resistance marker surveillance"
collection: publications
permalink: /publication/2018-04-01-Mars
excerpt: 'The recent advances in next-generation sequencing technologies provide a new and effective way of tracking malaria drug-resistant parasites. To take advantage of this technology, an end-to-end Illumina targeted amplicon deep sequencing (TADS) and bioinformatics pipeline for molecular surveillance of drug resistance in P. falciparum, called malaria resistance surveillance (MaRS), was developed. TADS relies on PCR enriching genomic regions, specifically target genes of interest, prior to deep sequencing. MaRS enables researchers to simultaneously collect data on allele frequencies of multiple full-length P. falciparum drug resistance genes (crt, mdr1, k13, dhfr, dhps, and the cytochrome b gene), as well as the mitochondrial genome. Information is captured at the individual patient level for both known and potential new single nucleotide polymorphisms associated with drug resistance. The MaRS pipeline was validated using 245 imported malaria cases that were reported to the Centers for Disease Control and Prevention (CDC). The chloroquine resistance crt CVIET genotype (mutations underlined) was observed in 42% of samples, the highly pyrimethamine-resistant dhps IRN triple mutant in 92% of samples, and the sulfadoxine resistance dhps mutation SGEAA in 26% of samples. The mdr1 NFSND genotype was found in 40% of samples. With the exception of two cases imported from Cambodia, no artemisinin resistance k13 alleles were identified, and 99% of patients carried parasites susceptible to atovaquone-proguanil. Our goal is to implement MaRS at the CDC for routine surveillance of imported malaria cases in the United States and to aid in the adoption of this system at participating state public health laboratories, as well as by global partners.'
date: 2018-04-01
venue: 'Antimicrobial Agents and Chemotherapy'
paperurl: 'https://journals.asm.org/doi/full/10.1128/aac.02474-17'
citation: 'Talundzic E, Ravishankar S, Kelley J, Patel D, Plucinski M, Schmedes S, Ljolje D, Clemons B, Madison-Antenucci S, Arguin PM, Lucchi NW. Next-generation sequencing and bioinformatics protocol for malaria drug resistance marker surveillance. Antimicrobial Agents and Chemotherapy. 2018 Apr;62(4):10-128.'
---
The recent advances in next-generation sequencing technologies provide a new and effective way of tracking malaria drug-resistant parasites. To take advantage of this technology, an end-to-end Illumina targeted amplicon deep sequencing (TADS) and bioinformatics pipeline for molecular surveillance of drug resistance in P. falciparum, called malaria resistance surveillance (MaRS), was developed. TADS relies on PCR enriching genomic regions, specifically target genes of interest, prior to deep sequencing. MaRS enables researchers to simultaneously collect data on allele frequencies of multiple full-length P. falciparum drug resistance genes (crt, mdr1, k13, dhfr, dhps, and the cytochrome b gene), as well as the mitochondrial genome. Information is captured at the individual patient level for both known and potential new single nucleotide polymorphisms associated with drug resistance. The MaRS pipeline was validated using 245 imported malaria cases that were reported to the Centers for Disease Control and Prevention (CDC). The chloroquine resistance crt CVIET genotype (mutations underlined) was observed in 42% of samples, the highly pyrimethamine-resistant dhps IRN triple mutant in 92% of samples, and the sulfadoxine resistance dhps mutation SGEAA in 26% of samples. The mdr1 NFSND genotype was found in 40% of samples. With the exception of two cases imported from Cambodia, no artemisinin resistance k13 alleles were identified, and 99% of patients carried parasites susceptible to atovaquone-proguanil. Our goal is to implement MaRS at the CDC for routine surveillance of imported malaria cases in the United States and to aid in the adoption of this system at participating state public health laboratories, as well as by global partners.

[Download paper here](https://journals.asm.org/doi/full/10.1128/aac.02474-17)

Recommended citation: Talundzic E, Ravishankar S, Kelley J, Patel D, Plucinski M, Schmedes S, Ljolje D, Clemons B, Madison-Antenucci S, Arguin PM, Lucchi NW. Next-generation sequencing and bioinformatics protocol for malaria drug resistance marker surveillance. Antimicrobial Agents and Chemotherapy. 2018 Apr;62(4):10-128.